server/src/service-layer/controllers/StripeWebhook.ts

```diff
@@ -1,38 +1,71 @@
 import { Controller, SuccessResponse, Route, Post, Request } from "tsoa"
 import Stripe from "stripe"
+import UserDataService from "data-layer/services/UserDataService"
+import AuthService from "business-layer/services/AuthService"
+import {
+  CHECKOUT_TYPE_KEY,
+  CheckoutTypeValues
+} from "business-layer/utils/StripeProductMetadata"
+import StripeService from "business-layer/services/StripeService"
 
 @Route("webhook")
 export class StripeWebhook extends Controller {
   @Post()
   @SuccessResponse(200, "Webhook post received")
   public async receiveWebhook(@Request() request: any): Promise<void> {
-    console.log("webhook received")
-
     const stripe = new Stripe(process.env.STRIPE_API_KEY)
-
+    // Ensure security of the endpoint by constructing an event
+    let event: Stripe.Event
     try {
-      const event: Stripe.Event = stripe.webhooks.constructEvent(
+      event = stripe.webhooks.constructEvent(
         request.rawBody,
         request.headers["stripe-signature"],
-        // process.env.STRIPE_LOCAL // test local api secret
-        process.env.STRIPE_API_SECRET
+        process.env.STRIPE_LOCAL
```

Comment:
can we update the workflows if we are going to change the name of env variables

Reply:
whoops sorry forgot to change that, should be STRIPE_API_SECRET, the LOCAL one is for local testing.